worker.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013, 2014, 2015 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+
+import logging
+from newsroom.flaskapp import Newsroom
+
+
+logger = logging.getLogger(__name__)
+app = Newsroom("newsroom_celery")
+celery = app.celery
```

Comment:
it would be better to have this file in newsroom module, so you can use it in newsroom-aap without making a copy of it